conda_recipes/isl-tc/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: isl-tc
```

Comment:
to make it specific to ISL, I think there was a comment long back when I was writing these scripts that we should make things as tc like ppcg-tc, etc so that we don't confuse with the other actual repos